frontend/lib/tests/build-pipeline.test.tsx

```diff
@@ -0,0 +1,43 @@
+import autobind from 'autobind-decorator';
+
+test("dynamic import works", () => {
+  return import('./build-pipeline-dynamic-import').then(mod => {
+    expect(mod.blah(5)).toEqual(6);
+  });
+});
+
+test('autobind decorator works', () => {
+  class Blarg {
+    constructor(readonly base: number) {}
+
+    @autobind
+    boop(x: number) {
+      return this.base + x;
+    }
+  }
+
+  const blarg = new Blarg(6);
+
+  expect(blarg.boop(1)).toEqual(7);
```

Comment:
Ack, we should really be detaching `boop` from `blarg` and calling that, to ensure that the autobinding actually works.